src/main/java/structures/Arrayer.java

```diff
@@ -41,8 +90,38 @@ public static int[] roundUp(double[] input) {
      * 
      * ex: [0, 4, -2, 17] -> 2 
      */
-    public static int findNegative(int[] input) {
-        return 0;
+    public static int findNegative(int[] input) { 
+
+       for (int i = 0; i < input.length; i++){
+        // loop created to keep track of positions
+            if (input [i] < 0) {
+                return i;
+            // if the number is negative, it will return the number's position
+        }
+       }
+        return -1;
+        // if all numbers are positive it will return -1 since there are no positions with a negative number
+        }
+
+     // accepts an array of ints and returns a new array containing only numbers in even positions   
+    public static int[] everyOther(int[] input){
+        
+        int evens = 0;
+        for (int i = 0; i < input.length; i++){
```

Comment:
-1 style, must include space between end of terminating condition and beginning of loop scope. ex: for (int i = 0; i < input.length; i++) { ...